src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -135,18 +135,18 @@ public static function register(self $handler = null, bool $replace = true): sel
         } else {
             $handlerIsRegistered = true;
         }
-        if (\is_array($prev = set_exception_handler([$handler, 'handleException'])) && $prev[0] instanceof self) {
+        if (\is_array($prev = set_exception_handler($handler->handleException(...))) && $prev[0] instanceof self) {
```

Comment:
I'm quite sure those `$prev[0] instanceof self` checks won't work anymore if you use a first-class callable instead of the array-based callables.

Reply:
You're right, I'm reverting all changes related to the ErrorHandler component anyway :+1: